src/dipdup/cli.py

```diff
@@ -183,6 +183,16 @@ async def _check_version() -> None:
         _logger.info(_skip_msg)
         return
 
+    from appdirs import user_cache_dir
+
+    cache_dir = Path(user_cache_dir('dipdup'))
+    cache_file = cache_dir / 'latest_version.json'
+
+    latest_version = await get_cached_version(cache_file)
```

Comment:
Installed DipDup version should be cached too so we can invalidate cache later when package will be updated.